css/main.css

```diff
@@ -325,3 +325,86 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.mu-contact-bottom {
+  display: inline;
+  float: left;
+  width: 100%;
+}
+
+#form-messages {
+  margin-bottom: 10px;
+  background-color: #fff;
+}
+
+.mu-contact-form .form-group input {
+  background: transparent;
+  border-radius: 8px;
+  color: #333;
+  font-size: 15px;
+  border: 1px solid #888;
+  height: 45px;
+  margin-bottom: 24px;
+}
+
+.mu-contact-form .form-group textarea {
+  background: transparent;
+  border: 1px solid #888;
+  border-radius: 8px;
+  color: #333;
+  font-size: 18px;
+  padding: 10px;
+  height: 190px;
+}
+
+.mu-contact-form .form-control:focus {
+  background-color: #fff;
+  outline: 0;
+  box-shadow: none;
+}
+© .mu-contact-form .form-control::-webkit-input-placeholder {
+  color: #888;
+  font-size: 15px;
+}
+
+.mu-contact-form .form-control:-moz-placeholder {
+  /* Firefox 18- */
+  color: #888;
+  font-size: 15px;
+}
+
+.mu-contact-form .form-control::-moz-placeholder {
+  /* Firefox 19+ */
+  color: #888;
+  font-size: 15px;
+}
+
+.mu-contact-form .form-control:-ms-input-placeholder {
+  color: #888;
+  font-size: 15px;
```

Comment:
- [ ] *[Required]* Your CSS code should be DRY (Don't Repeat Yourself). Unfortunately, you have so many code repetitions here. `color: #888` and `font-size: 15px` is being repeated 4 times. The illustration below should help:
```css
one,
two,
three {
    color: #eee;
    background-color: #abc;
}
```

Reply:
Done!